mgtool/file.go

```diff
@@ -123,6 +123,7 @@ func (s *fileState) handleFileStream(inFile io.Reader, filename string) error {
 		defer out.Close()
 
 		// Write the body to file
+
```

Comment:
random new line?

Reply:
Autofixed by linter - probably because the comment above is taken as a "heading", let me see if this can be fixed